LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2023 Chan Zuckerberg Initiative
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
Why change this back?

Reply:
If you read the text above the copyright notice, I think this is intended as a template. This also explains why even Apache projects leave it like that (e.g. [kafka](https://github.com/apache/kafka/blob/trunk/LICENSE)).

Based on the same text, I think actual copyright notice referencing the owner/licensor should appear in each source code file or maybe even every file that's distributed.

Also, I'd prefer to do all the licenses at once and consistently, rather than make this a special case.